### DISCUSSION TITLE
OpenBiBi, a giscus powered chrome extension, enable you comment any webpage.

### DISCUSSION THREAD

**Discussion by FachengG** [giscus/giscus#1185] [Ideas]
OpenBiBi, a giscus powered chrome extension, enable you comment any webpage. 

Checkout it here: https://chrome.google.com/webstore/search/openbibi

Need your input for improvement! Thank you @laymonage 

Some Idea in my mind:
Internal Tool plan: Thinking about build it a internal tool for company (probably SaaS company, whose products are webpage). An addition to slack/jira. 
Public plan: Make it like discord, multiple server, people can chat on whatever they want. Comment only share within server.

**Comment by laymonage** [comment]
Hi, thank you for building this. It's an interesting idea!

However, would you please use a self-hosted version of giscus instead of the https://giscus.app service directly? I can imagine that this will add a considerable amount of load to our servers, and given that we're running the service on free tiers and sponsorship given by cloud platforms, I'd like to keep the load to a minimum. Just this morning, I had an email from Supabase saying that we've surpassed the 5GB bandwidth and will need to start paying $25/mo (or switch to a different solution).

Thanks for making this and I hope you understand.

**Reply by FachengG** [reply]
Thanks for your reply. I am working on building backend server for the OpenBiBi app. In the mean time (sadly) it does not have any traffics at all and only used for experimental/demo purpose. I believe before it gain any traffic, my backend will be ready for it.
